consensus/polybft/consensus_runtime.go

```diff
@@ -237,16 +237,16 @@ func (c *consensusRuntime) OnBlockInserted(block *types.Block) {
 	// after the block has been written we reset the txpool so that the old transactions are removed
 	c.config.txPool.ResetWithHeaders(block.Header)
 
-	// handle commitment and proofs creation
-	if err := c.stateSyncManager.PostBlock(&PostBlockRequest{Block: block}); err != nil {
-		c.logger.Error("failed to post block state sync", "err", err)
-	}
-
 	var (
 		epoch = c.epoch
 		err   error
 	)
```

Comment:
Don't see any benefit in moving this var block. If we are going to move it, I'd just put it at the very beginning of function (below locking).

Reply:
I don't think this PR changed anything about this var block :confused: . I think it was added at this place because we don't need those variables before the given lines.